### DISCUSSION TITLE
Separate directory for products

### DISCUSSION THREAD

**Discussion by matejak** [ComplianceAsCode/content#7070] [Ideas]
The number of products that this content project supports is growing, and the directory structure is becoming a mess. We have a PR that aims to solve this - #7018 that moves products to a dedicated directory. So far people seem to like it, but it is a change that moves a large amount of files, and we feel that the discussion hasn't been proportional to the impact of the PR.
The change breaks compatibility of patches, but one can easily create a set of symlinks that act as a compatibility layer, so we think that the impact shouldn't be too bad to anybody either - let us know if we are wrong.
So please go and check #7018, and leave us some comments!

**Comment by matejak** [comment]
The PR has landed. It accidentally got to the master branch by non-standard means, but it technically passed the code review.
It can lead to conflicts in PRs, but `git rebase` should handle them without any need to manually edit files.

**Reply by cipherboy** [reply]
Thanks @matejak!
